test/selenium/util/PayPalCheckout.scala

```diff
@@ -19,7 +19,11 @@ class PayPalCheckout(implicit val webDriver: WebDriver) extends Browser {
 
   def logIn(): Unit = clickOn(loginButton)
 
-  def acceptPayment(): Unit = clickOn(agreeAndPay)
+  def acceptPayment(): Unit = {
+    pageHasElement(agreeAndPay)
```

Comment:
I would re-name _payPalHasPaymentSummary_ to _payPalSummaryHasLoaded_ and add the elementIsClickable check to that method

Reply:
Good idea - its a better fit there.